src/providers/SpeechProvider/tts.js

```diff
@@ -68,10 +68,21 @@ const initElevenLabsSynthesizer = () => {
   }
 };
 
-const ensureElevenLabsInitialized = () => {
+const ensureElevenLabsInitialized = async () => {
   if (!elevenLabsSynthesizer || !elevenLabsSynthesizer.isInitialized()) {
     initElevenLabsSynthesizer();
```

Comment:
this function should return the new eleven labs instance. In that way the code will be more readable. 

```js
elevenLabsSynthesizer = initElevenLabsSynthesizer();
```

Reply:
See the following https://github.com/magush27/cboard/pull/1